apidef/oas/security_test.go

```diff
@@ -418,19 +418,13 @@ func TestOAS_OAuth(t *testing.T) {
 						securityName: &oauth,
 					},
 				},
-				GatewayTags: &GatewayTags{
-					Enabled: true,
-					Tags:    []string{},
-				},
 			},
 		},
 	}
 
 	var api apidef.APIDefinition
 	oas.ExtractTo(&api)
 
-	assert.False(t, api.TagsDisabled)
```

Comment:
This tests a default assertion. Unless we change the default, and it's not likely, the test verifies it.